t/api.t

```diff
@@ -31,6 +31,29 @@ my $t              = Test::Mojo->new(Dashboard => $config);
 $dashboard_test->no_fixtures($t->app);
 my $auth_headers = {Authorization => 'Token test_token', Accept => 'application/json'};
 
+sub _incident_json {
```

Comment:
suggestion(non-blocking): I know that all 3 caller use the same number and variable, but the function name is quite general I think, maybe something like `_incident_json_16860` should be more specific and if we need a generalization later on we can set up args and change the sign of the function. Are too much steps in case of refactoring?

Also, why create a function to return an hash and not create a const hash at the start of the test?

Reply:
Extracted a const hash.